### DISCUSSION TITLE
img2img Resize Mode

### DISCUSSION THREAD

**Discussion by kklouzal** [Kilvoctu/aiyabot#195] [Ideas]
Requestion feature to set the 'Resize Mode' on img2img generations. A default /setting for this would be nice as well.

**Comment by Kilvoctu** [comment]
Heya, not sure what resize mode is, but I added this request to the projects list for consideration.

**Reply by kklouzal** [reply]
The Automatic1111 web interface, when going to the img2img tab, gives you a few different resize modes:
![image](https://github.com/Kilvoctu/aiyabot/assets/2928845/b57b10b4-50e8-43a4-aa85-c9dfde850c1c)
The crop and resize, or resize and fill modes make non-uniform images (very wide or very tall) images work A LOT better when doing an img2img so it's almost a necessity for users to be able to choose between these different modes, otherwise the generated output images will be stretched and skewed.

https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Features#resizing
